### DISCUSSION TITLE
Dry Powder Spraying Model

### DISCUSSION THREAD

**Discussion by MaoyiHe** [firemodels/fds#13780] [Q&A]
Hello，Is there an expert who can help me analyze whether the third line predefines sodium bicarbonate or if it is just a comment? In the spray extinguishing model, is the extinguishing substance water vapor or sodium bicarbonate?
[1.txt](https://github.com/user-attachments/files/17829118/1.txt)

**Comment by drjfloyd** [comment]
If you run the input, does it appear in the the .out file summary of species?

**Comment by MaoyiHe** [comment]
I checked the document and found that there is no code related to sodium bicarbonate. The author mentioned that the underlying code of FDS is written in Fortran and that sodium bicarbonate particles have been embedded in it. It is said that sodium bicarbonate will appear during the spraying process. Can this be verified?  This is the .out file

[formal10.out.txt](https://github.com/user-attachments/files/17830968/formal10.out.txt)

**Reply by drjfloyd** [reply]
I suggest you carefully read the User's Guide to learn how to define gas species and solid materials for particles.
